packages/settings/src/lib/photon.ts

```diff
@@ -210,8 +210,9 @@ const organizationSettings: {
     ...defaultSettings,
     logo: 'sana_care_logo.png',
     accentColor: '#FBCC45',
-    mailOrderNavigate: false,
+    mailOrderNavigate: true,
     mailOrderNavigateProviders: [AMAZON_PHARMACY_ID],
+    enableCourierNavigate: true,
```

Comment:
Can we do this in a separate PR I think, just in case this doesnt work for some reason and we want to walk it up to photon